pkg/apis/serving/k8s_validation_test.go

```diff
@@ -647,10 +734,11 @@ func TestPodSpecValidation(t *testing.T) {
 
 func TestPodSpecMultiContainerValidation(t *testing.T) {
 	tests := []struct {
-		name    string
-		ps      corev1.PodSpec
-		cfgOpts []configOption
-		want    *apis.FieldError
+		name     string
+		ps       corev1.PodSpec
+		cfgOpts  []configOption
+		want     *apis.FieldError
+		errLevel apis.DiagnosticLevel
```

Comment:
question: is the default value (0) == apis.ErrorLevel


looks like it is - https://github.com/knative/pkg/blob/247510c00e9dbb7f5a23b76e8ab76654de173a04/apis/field_error.go#L38

Reply:
I think it would be more clear in the tests that don't assert warnings to just filter in the `t.Run` instead of adding the additional option to the test case

```
got = got.Filter(apis.ErrorLevel)
```

Reply:
Yes, the default value is `error` for compatibility with all the existing code.

I think I was trying to be consistent here, but I can roll all these back if you prefer.